k2/csrc/log.h

```diff
@@ -174,6 +174,15 @@ inline bool EnableCudaDeviceSync() {
   return enable_cuda_sync;
 }
 
```

Comment:
I guess user can disable all checks for `RaggedShape` with it, but maybe better with documentation.